lib/class-wp-rest-menu-items-controller.php

```diff
@@ -181,19 +203,35 @@ public function create_item( $request ) {
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
 	public function update_item( $request ) {
-		$valid_check = $this->get_nav_menu_item( $request['id'] );
+		$prepared_nav_item = $this->update_item_validate( $request );
```

Comment:
Aside from making the methods shorter why did the update(delete)_item methods get split in two?

Reply:
To make it possible to validate the entire batch first, and only save if it's valid